### PR TITLE
MOTS-317 Remove peer supervisor field

### PR DESCRIPTION
**Pull request by kkaczmarczyk**

### CODE REVIEW

**Comment by kkaczmarczyk** [comment]
@awalkowiak  can you whether mobile part looks OK?

File: mobile/src/components/HealthWorkersForm.js [hunk 1]
```diff
@@ -343,7 +332,6 @@ function mapStateToProps(state) {
     districtId: selector(state, 'districtId'),
     chiefdomId: selector(state, 'chiefdomId'),
     facilityId: selector(state, 'facilityId'),
-    hasPeerSupervisor: selector(state, 'hasPeerSupervisor'),
```
Comment (pmuchowski) [inline mobile/src/components/HealthWorkersForm.js]: you should also remove it from propTypes, defaultProps and FormField

File: src/webapp/js/components/health-workers-form.js [hunk 2]
```diff
@@ -233,7 +218,6 @@ function mapStateToProps(state) {
     districtId: selector(state, 'districtId'),
     chiefdomId: selector(state, 'chiefdomId'),
     facilityId: selector(state, 'facilityId'),
-    hasPeerSupervisor: selector(state, 'hasPeerSupervisor'),
```
Comment (pmuchowski) [inline src/webapp/js/components/health-workers-form.js]: as above

**Comment by kkaczmarczyk** [comment]
fixed

**Pull request merged**